ingestclient/core/backend.py

```diff
@@ -30,6 +30,48 @@
 from ..utils.backoff import get_wait_time
 
 
+class IngestStatus(Enum):
+    """Return values used by upload()."""
+    STOP = 0            # Stop ingest.
+    UPLOAD = 1          # Continue uploading.
+    WAIT = 2            # Wait on the backend to finish something.
+    COMPLETING = 3      # Start polling job status while the backend verifies job.
+
+
+class BackendStatus(Enum):
+    """This enum mirrors boss.git/django/bossingest/models.py"""
+    PREPARING = 0
```

Comment:
I often start Enums at 1, with the idea that you can then use `if MyEnum.ANY` as truthy. Prevents accidental collision with 0. maybe unnecessary here tho

Reply:
This matches the values used in the DB. I guess it could be transformed, but I thought it'd be safer to just match what's used in the backend.

Reply:
agreed, resolving this thread :)